### PR TITLE
Docs: update trash.md

### PR DESCRIPTION
**Pull request by yuhr123**

### CODE REVIEW

File: docs/zh_cn/security/trash.md [hunk 1]
```diff
@@ -102,9 +102,17 @@ juicefs restore $META_URL 2023-08-14-05 --put-back
...
-因此，为了让过期文件能够正常清理，需要至少 1 个在线的挂载点，并且能够正常执行后台任务（不能开启 [`--no-bgjob`](../reference/command_reference.md#mount)）。从回收站彻底删除文件时，会同步删除元数据和对象存储，再加上清理任务由客户端每小时定期执行，因此面对大量文件过期（或者强制删除）时，对象存储的清理速度未必和你期望的一样快。如果希望更快速删除过期文件，可以挂载多个挂载点来突破单个客户端的删除速度上限，然后在这些挂载点的 `.trash` 目录下，对需要删除的文件按批次手动运行 `juicefs rmr` 来强行删除。
+如果你希望在过期时间到来之前彻底删除文件，可以用 [`juicefs rmr`](../reference/command_reference.md#rmr) 或系统自带的 `rm` 命令来删除回收站目录 `.trash` 中的文件，这样就能立刻释放存储空间。
+
+例如，彻底删除回收站中某个目录：
+
+```shell
+juicefs rmr .trash/2022-11-30-10/
+```
+
+另外，回收站自动清理依赖 JuiceFS 客户端的后台任务，为了保证后台任务能够正常执行，需要至少 1 个在线的挂载点，并且在挂载文件系统时不可以使用 [`--no-bgjob`](../reference/command_reference.md#mount) 参数。如果希望更快速删除过期文件，可以挂载多个挂载点来突破单个客户端的删除速度上限。
```
Comment (SandyXSD) [inline docs/zh_cn/security/trash.md:115]: move this paragraph up, above the manual clean-up?

File: docs/zh_cn/security/trash.md [hunk 2]
```diff
@@ -102,9 +102,17 @@ juicefs restore $META_URL 2023-08-14-05 --put-back
 
 ## 彻底删除文件 {#purge}
 
-回收站目录各方面表现都和普通目录一致——恢复文件就是 `mv` 出来，如果想彻底删除文件，那么 `rm` 就能做到（需要 root 权限）。但要注意，**就算回收站内的文件已经到达了过期时间，也未必会立刻从回收站消失**，这是因为过期文件的清理由 JuiceFS 客户端定期在后台任务（background job，也称 bgjob）执行，默认每小时清理一次。
+当回收站中的文件到了过期时间，会被自动清理。需要注意的是，文件清理由 JuiceFS 客户端的后台任务（background job，也称 bgjob）执行，默认每小时清理一次，因此面对大量文件过期时，对象存储的清理速度未必和你期望的一样快，可能需要一些时间才能看到存储容量变化。
 
-因此，为了让过期文件能够正常清理，需要至少 1 个在线的挂载点，并且能够正常执行后台任务（不能开启 [`--no-bgjob`](../reference/command_reference.md#mount)）。从回收站彻底删除文件时，会同步删除元数据和对象存储，再加上清理任务由客户端每小时定期执行，因此面对大量文件过期（或者强制删除）时，对象存储的清理速度未必和你期望的一样快。如果希望更快速删除过期文件，可以挂载多个挂载点来突破单个客户端的删除速度上限，然后在这些挂载点的 `.trash` 目录下，对需要删除的文件按批次手动运行 `juicefs rmr` 来强行删除。
+如果你希望在过期时间到来之前彻底删除文件，可以用 [`juicefs rmr`](../reference/command_reference.md#rmr) 或系统自带的 `rm` 命令来删除回收站目录 `.trash` 中的文件，这样就能立刻释放存储空间。
```
Comment (SandyXSD) [inline docs/zh_cn/security/trash.md:107]: both of these two commands require root privilege.

Comment (yuhr123) [reply docs/zh_cn/security/trash.md]: done

File: docs/zh_cn/security/trash.md (hunk 1 above)
Comment (yuhr123) [reply docs/zh_cn/security/trash.md]: done

**Pull request merged**